check_critical_css.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+folder="build/critical-css/"
+file_list=$(find "$folder" -name '*.css')
```

Comment:
Is the `find` relative to the script or the cwd?

Reply:
always to the cwd

Reply:
(bash does not have a `__DIR__` and there are obscure idioms to achieve that)

Reply:
Looking at the other scripts here I thought we'd catered for running them from elsewhere, but we haven't (remember it being done in some projects).

Reply:
builder: https://github.com/elifesciences/builder/blob/master/bldr#L3-L4

Reply:
I'm probably thinking of https://github.com/elifesciences/elife-website